Tests/AutomergeTests/TestHistory.swift

```diff
@@ -0,0 +1,29 @@
+import Automerge
+import XCTest
+
+class HistoryTests: XCTestCase {
+    func testChangeCountsInHeads() {
+        let doc = Document()
+        try! doc.put(obj: ObjId.ROOT, key: "key", value: .String("one"))
+
+        var heads = doc.heads()
+        XCTAssertEqual(heads.count, 1)
+        heads = doc.heads()
+        try! doc.put(obj: ObjId.ROOT, key: "key", value: .String("two"))
+        XCTAssertEqual(heads.count, 2)
```

Comment:
This should still be one I think. There are multiple heads when there are concurrent changes, for example if you were to fork the document then make a change in the fork, then merge the fork back into the original document you would have two heads. Then if you were to make another change in the original document you would be back to one head (which has the previous two heads as it's dependencies).

Reply:
That explains why my test is failing ;-)